### DISCUSSION TITLE
Problem with handshake capture on gl.inet gl-ar300m16-ext

### DISCUSSION THREAD

**Discussion by dyba404** [xchwarze/wifi-pineapple-cloner#2] [General]
Problem with handshake capture on gl.inet gl-ar300m16-ext:

Hi, I have a problem with my gl.inet gl-ar300m16-ext device. I have installed the firmware gl-ar300m-universal-sysupgrade.bin, which is compatible with my wifi adapter with the mt7612u chip. The adapter works correctly to do deauth and recon, but it fails to capture the handshake of the wifi networks. I have tried with several networks and none of them give me the handshake. Does anyone know how to solve this problem? Is it a firmware or adapter problem? I appreciate any help or suggestions. Thank you.

**Comment by xchwarze** [comment]
Did you do lab tests in a controlled environment?
Maybe it's something that's even malfunctioning in the real pineapple.

**Reply by dyba404** [reply]
Thanks for your reply. Yes, I tested in a controlled environment with my own router and a client device. The wifi adapter manages to deauth to the client, but does not capture the handshake that is generated when reconnecting. I have checked the firmware and adapter settings, but can't find the problem. do you have any idea what could be going wrong?

**Reply by xchwarze** [reply]
As far as I remember, this is done by pineap. It would be necessary to find out if the original version has the same defect.

Maybe as a starting point it would be a good idea to test with several other adapters and if the defect is still present to rule things out you could also build with the same version that uses the original release which is the .02. That version works very badly everything that is 5g so you would have to repeat the tests with common adapters and not dual.

**Reply by HatoSec** [reply]
Did you ever figure this out?
